crates/neon/src/sys/bindings/functions.rs

```diff
@@ -51,6 +51,9 @@ mod napi1 {
 
             fn close_handle_scope(env: Env, scope: HandleScope) -> Status;
 
+            fn instanceof(env: Env, object: Value, constructor: Value, result: *mut bool)
```

Comment:
This is awesome! Thanks for adding it. I think we should add it as a method on the `Value` trait.

```rust
trait Value {
    fn instance_of<V: Value>(&self, cx: &mut Cx, ctor: &V) -> bool {
        /* ... */
    }
}
```

Reply:
Would you mind opening a separate PR to implement this? It's a great feature and we can get it merged pretty quick.

I'm happy to pick it up if you prefer.

Reply:
Done, I've also added a dedicated test